api/client/webclient/webclient_test.go

```diff
@@ -26,6 +30,88 @@ import (
 	"github.com/gravitational/teleport/api/defaults"
 )
 
+func newPingHandler(path string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		if req.RequestURI != path {
+			w.WriteHeader(http.StatusNotFound)
+			return
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		json.NewEncoder(w).Encode(PingResponse{ServerVersion: "test"})
+	})
+}
+
+func TestPingHttpFallback(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	handler := newPingHandler("/webapi/ping")
+	httpSvr := httptest.NewServer(handler)
+	defer httpSvr.Close()
+
+	t.Run("Allowed on insecure & loopback", func(t *testing.T) {
+		_, err := Ping(ctx, httpSvr.Listener.Addr().String(), true, nil, "")
```

Comment:
```suggestion
		_, err := Ping(ctx, httpSvr.Listener.Addr().String(), true /* insecure */, nil /* pool */, "" /* connectorName */)
```

Same for calls below.

Reply:
done